test/lib/ufe/testMoveCmd.py

```diff
@@ -394,5 +394,37 @@ def createCommonAPI(testCase, sphereXformable):
 
         self.runTestOpUndo(createCommonAPI, 'xformOp:translate')
 
+    def testBadSceneItem(self):
```

Comment:
Test Driven Development: wrote the test first, it crashed and failed, wrote the fix, test passes.